tests/unit/test_loader_collection.py

```diff
@@ -101,6 +101,13 @@
    "chksum_sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
    "format": 1
   },
+  {
+   "name": "CHANGELOG.rst",
```

Comment:
awesome, is there anyway to find that out etc? saw some being spat out in the logs but wasn't sure if there was a way I could see it before as to prevent failures etc

Reply:
answered my own question, please ignore